alerting/build.gradle

```diff
@@ -88,61 +82,74 @@ dependencyLicenses.enabled = false
 validateNebulaPom.enabled = false
 thirdPartyAudit.enabled = false
 
+loggerUsageCheck.enabled = false
+licenseHeaders.enabled = false
+
 def es_tmp_dir = rootProject.file('build/private/es_tmp').absoluteFile
 es_tmp_dir.mkdirs()
 
 test {
     systemProperty 'tests.security.manager', 'false'
 }
 
-def _numNodes = findProperty('numNodes') as Integer ?: 1
-integTest {
-    runner {
-        systemProperty 'tests.security.manager', 'false'
-        systemProperty 'java.io.tmpdir', es_tmp_dir.absolutePath
-
-        systemProperty "https", System.getProperty("https")
-        systemProperty "user", System.getProperty("user")
-        systemProperty "password", System.getProperty("password")
-
-        // The 'doFirst' delays till execution time.
-        doFirst {
-            // Tell the test JVM if the cluster JVM is running under a debugger so that tests can
-            // use longer timeouts for requests.
-            def isDebuggingCluster = getDebug() || System.getProperty("test.debug") != null
-            systemProperty 'cluster.debug', isDebuggingCluster
-            // Set number of nodes system property to be used in tests
-            systemProperty 'cluster.number_of_nodes', "${_numNodes}"
-            // There seems to be an issue when running multi node run or integ tasks with unicast_hosts
-            // not being written, the waitForAllConditions ensures it's written
-            getClusters().forEach { cluster ->
-                cluster.waitForAllConditions()
-            }
-        }
 
-        // The --debug-jvm command-line option makes the cluster debuggable; this makes the tests debuggable
-        if (System.getProperty("test.debug") != null) {
-            jvmArgs '-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005'
-        }
-    }
+import org.elasticsearch.gradle.test.RestIntegTestTask
```

Comment:
imports should be at the top

Reply:
+1

Reply:
meant to be clear for this task, move to top now